File: week2/assignment/anagram.java
import java.util.Arrays;


public class anagram {
    public static void main(String[] args) {
        Boolean a = isAnagram("art", "rat");
        System.out.println(a);
    }

    public static Boolean isAnagram(String s, String t) {
        char[] arr1 = s.toCharArray();
        Arrays.sort(arr1);
        s = new String(arr1);

        char[] arr2 = t.toCharArray();
        Arrays.sort(arr2);
        t = new String(arr2);
        
        System.out.println();
        System.out.println();
        System.out.println(s);
        System.out.println(t);

        if(!t.equals(s)) {
            return false;
        }
        else {
            return true;
        }
    }
}

File: week2/assignment/anagram.js
function isAnagram(s, t) {
    if (s.length !== t.length) return false;
    const sorted1 = s.split('').sort().join('');
    const sorted2 = t.split('').sort().join('');
    
    return sorted1 === sorted2;
}

s = "anagram";
t = "bolaomj";

console.log(isAnagram(s, t));